htmresearch/algorithms/column_pooler.py

```diff
@@ -543,7 +543,8 @@ def _rightVecSumAtNZGtThreshold_sparse(sparseMatrix,
   an eventual C implementation would.
   """
   denseArray = numpy.zeros(sparseMatrix.nCols(), dtype=GetNTAReal())
-  denseArray[sparseBinaryArray] = 1
+  if len(sparseBinaryArray) > 0:
```

Comment:
And it'd be more correct, since any tuple will cause problems, not just an empty tuple.